src/__experimental__/components/radio-button/radio-button-group.component.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import tagComponent from '../../../utils/helpers/tags';
+import Label from '../label';
+import { StyledRadioButtonGroup } from './radio-button.style';
+
+function initialTabIndex(index) {
+  return index ? -1 : 0;
```

Comment:
I think something like `childIndex` would be a better name than `index` for this, to make it clear that it's referring to a child element's index (not a `tabindex`).

Since it's a `number` (not a `boolean`), I think it'd also be good to make the condition more explicit (instead of relying on [JavaScript's truthiness rules](https://developer.mozilla.org/en-US/docs/Glossary/Truthy)):

```js
return (childIndex > 0) ? -1 : 0;
```